src/components/checkboxes/index.md.njk

```diff
@@ -65,6 +65,24 @@ You can add hints to checkbox items to provide additional information about the
 
 {{ example({group: "components", item: "checkboxes", example: "hint", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+### Add an option for ‘none’
+
+When 'none' would be a valid answer, give users the option to check a box to say none of the other options apply to them. Without this option, users would have to leave all of the boxes unchecked. This requirement also makes sure users do not skip the question by accident.
```

Comment:
Suggestion to make the "if not" statement a bit clearer.

```suggestion
When 'none' would be a valid answer, give users the option to check a box to say none of the other options apply to them — without this option, users would have to leave all of the boxes unchecked. Giving this option also makes sure users do not skip the question by accident.
```

Reply:
@calvin-lau-sig7  I like it! Minor tweak on your tweak:

```When 'none' would be a valid answer, give users the option to check a box to say none of the other options apply to them — without this option, users would have to leave all of the boxes unchecked. Giving users this option also makes sure they do not skip the question by accident.```